pkgs/tools/system/netdata/default.nix

```diff
@@ -59,6 +59,12 @@ in stdenv.mkDerivation rec {
     # Avoid build-only inputs in closure leaked by configure command:
     #   https://github.com/NixOS/nixpkgs/issues/175693#issuecomment-1143344162
     ./skip-CONFIGURE_COMMAND.patch
+    
+    # Fix stream authentication: https://github.com/netdata/netdata/pull/14063
+    ./fix-stream-authentication.patch
+
+    # Fix alarm notification commands: https://github.com/netdata/netdata/pull/14064
+    ./fix-alarm-notification-commands.patch
```

Comment:
Please use `fetchpatch` with the merged upstream commit patches here, if they apply cleanly.